app/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/GetRecordsConnector.scala

```diff
@@ -47,18 +47,13 @@ class GetRecordsConnector @Inject() (
       if (isHawk) s"${appConfig.hawkConfig.getRecordsUrl}/$eori/$recordId"
```

Comment:
we could also get read of this if statement if we pass in the url instead of a boolean. We can talk about if you want.